docs/usage/closed-prs.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Closed PRs
+description: How Renovate handles closed PRs
+---
+
+# Ignoring closed PRs
+
+By default renovate will ignore closed PRs and won't raise any PRs for same versions.
+If you want renovate to ignore closed PRs, then make the following changes to the closed PR -
```

Comment:
These sentences seem to contradict each other. Maybe don't use the word "ignore" in both

Reply:
I think they're trying to explain that you can tell Renovate to ignore the PR in two ways:

- Rename PR
- Attach "ignore label" to the PR

But it's not worded correctly right now.